airbyte-webapp/src/locales/en.json

```diff
@@ -510,6 +518,10 @@
   "settings.account": "Account",
   "settings.accountSettings.updateEmailSuccess": "Email updated",
   "settings.cookiePreferences": "Cookie Preferences",
+  "settings.dataResidency": "Data Residency",
+  "settings.defaultDataResidency": "Default Data Residency",
+  "settings.defaultGeography": "Geography",
+  "settings.defaultDataResidencyDescription": "Choose the default preferred data processing location for all of your connections. Setting this choice means your data never leaves this region. <lnk>Learn more</lnk>.",
```

Comment:
This is strictly spoken not completely true, and I feel this might be the place we need to clarify about that. Actually _some_ data leaves the region. If you have incremental syncs the cursor field values, which we'll store as a "state" for the next run are always stored in our control plane (such US), no matter where the connection ran. Meaning if you chose a field as a cursor field that would anyhow have sensitive data in it (not entirely sure of a good example for that), this data will be send and stored in the US.

cc @andyjih we talked about that a while back, and I think it would be good if you could give your input on wording and where we might want to document that?

Reply:
Agreed, it's a little misleading. @andyjih also wanted to add a short clarification that this default only affects new connections. How about the following?

> Choose the default preferred data processing location for all of your connections. The default Data Residency setting only affects new Connections. Existing Connections will retain their Data Residency setting. \<lnk\>Learn more\</lnk\>.

Reply:
Edit: looks like the docs PR also has some discussion on this: https://github.com/airbytehq/airbyte/pull/18872#discussion_r1013643704